src/components/Buttons/SubmitButton.jsx

```diff
@@ -14,7 +15,22 @@ const loadingColors = {
     danger: "!bg-danger-light",
 };
 
-export const SubmitButton = ({ color = "primary", isLoading, className, children, ...rest }) => {
+export const SubmitButton = ({ color = "primary", isLoading, isSuccess, className, children, ...rest }) => {
+    const [showSuccess, setShowSuccess] = useState(false);
+
+    useEffect(() => {
+        if (isSuccess && !isLoading) {
+            setShowSuccess(true);
+            setTimeout(() => setShowSuccess(false), 3000);
```

Comment:
```suggestion
            const timer = setTimeout(() => setShowSuccess(false), 3000);
```

Then you need to:

```
return () => clearTimeout(timer);
```

So things are cleaned up. Google useEffect with timer